MidasCivil_Adapter/CRUD/Read/Results/MeshResults.cs

```diff
@@ -98,12 +98,60 @@ private IEnumerable<IResult> ExtractMeshForce(List<int> ids, List<string> loadca
 
         private IEnumerable<IResult> ExtractMeshStress(List<int> ids, List<string> loadcaseIds, MeshResultLayer meshResultLayer)
         {
-            return null;
+            /***************************************************/
+            string filePath = directory + "\\Plate Stress(L).xls";
+            string csvPath = ExcelToCsv(filePath);
+            List<string> meshStressText = File.ReadAllLines(csvPath).ToList();
+            List<MeshStress> meshStresses = new List<MeshStress>();
+
+            List<string> meshStress = meshStressText[10].Split(',').ToList();
+            List<string> InitialmeshStress = meshStress;
+            meshStresses.Add(Convert.ToMeshStress(meshStress));
+
+            for (int i = 11; i < meshStressText.Count; i++)
+            {
+                meshStress = meshStressText[i].Split(',').ToList();
+                for (int j = 0; j < meshStress.Count; j++)
+                {
+                    if (meshStress[j] == "")
+                    {
+                        meshStress[j] = InitialmeshStress[j];
+                    }
+                }
```

Comment:
This loop is a little wasteful as there are only three properties to inherit from the previous entry (ElementID, Loadcase and Node).  Also, you are inheritng from `InitialmeshStress`, which never gets updated, so the ElementID, Loadcase and Node are all inherited from the first entry, instead of the entry directly above it.